addons/website_slides/models/slide_channel.py

```diff
@@ -517,21 +522,25 @@ def message_post(self, *, parent_id=False, subtype_id=False, **kwargs):
     # Business / Actions
     # ---------------------------------------------------------
 
-    def action_redirect_to_members(self, state=None):
+    def action_redirect_to_members(self, completed=False):
+        """
+               Redirects to attendees of the course. If completed is True, a filter will be
+               added in action that will display only attendees who have completed the course.
+        """
+        self.ensure_one()
         action = self.env["ir.actions.actions"]._for_xml_id("website_slides.slide_channel_partner_action")
-        action['domain'] = [('channel_id', 'in', self.ids)]
-        if len(self) == 1:
-            action['display_name'] = _('Attendees of %s', self.name)
-            action['context'] = {'active_test': False, 'default_channel_id': self.id}
-        if state:
-            action['domain'] += [('completed', '=', state == 'completed')]
+        action['display_name'] = _('Attendees of %s', self.name)
+        action_ctx = {'search_default_channel_id': self.ids, 'active_test': False}
+        if completed:
+            action_ctx.update({'search_default_completed': 1})
+        action['context'] = action_ctx
         return action
 
     def action_redirect_to_running_members(self):
         return self.action_redirect_to_members('running')
```

Comment:
This method needs to be removed as previously explained since it will no longer be working (and seems to not be called anywhere).